1_fetch/src/download.R

```diff
@@ -1,8 +1,7 @@
-library(sbtools)
 
-download_data <- function(output_dir = file.path('1_fetch', 'out')) {
-  dir.create(output_dir, showWarnings = FALSE)
+download_data <- function(out_filepath = file.path('1_fetch', 'out', 'model_RMSEs.csv')) {
```

Comment:
Nice work passing in the out_filepath as an argument, so that anyone who is looking through `_targets.R` can see what the output file is named and where it is saved. Can you add additional arguments so that we could potentially use this same function to download any specified file from any sb id?

Reply:
Sounds good! I made this change in 09c6230. I also changed the function name to `download_sb_file`, and I removed the default value for the `out_filepath` argument - it didn't seem to make sense to have such a specific default file path if the function is to be reused.